src/zc/buildout/__init__.py

```diff
@@ -13,9 +13,108 @@
 ##############################################################################
 """Buildout package
 """
+from setuptools.extern.six.moves import urllib
+from setuptools.package_index import PackageIndex
+from setuptools.package_index import URL_SCHEME
+from setuptools.package_index import HREF
+from setuptools.package_index import distros_for_url
+from setuptools.package_index import htmldecode
```

Comment:
We have a requirement of setuptools >= 8.0.

setuptools 8.0 does not have setuptools.extern.

Reply:
Right, I forgot to update minimal setuptools requirement.
I am eager to move to recent setuptools like 41.3.0 (chosen because it is the last one that does not complain about 2.7 if I remember well). What would be your opinion ?

Reply:
I don't do many buildout-based deployments nowadays, but in the past I'd decided that the strategy of always upgrading setuptools to the latest version was the best way to avoid problems.  So, I'm fine with depending on whatever.

Other buildout users might have different opinions.

Reply:
I think only `pip` complains about Python 2.7, not `setuptools`.
Version 42 deprecates easy_install.
But 41.3 sounds like roughly a fine number to have as minimum.

Reply:
You are right, I mixed complaining about 2.7 and `easy_install` deprecation.

I will go for last before 42: 41.6.0